extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/handler/AstroThingHandler.java

```diff
@@ -113,8 +124,11 @@ public void dispose() {
             schedulerFuture.cancel(true);
             schedulerFuture = null;
         }
-        stopJobs();
-        quartzScheduler = null;
+        if (scheduledExecutor.isPresent()) {
```

Comment:
`stopJobs()` method is still there with a different implementation but here stopping the scheduler will stop the scheduled jobs and the scheduler will be shutdown. On the other hand, according to my implementation of stopJobs(), the scheduler will still be active but all the scheduled jobs will be removed.